packages/bundler-security/src/metroSecureResolver.js

```diff
@@ -0,0 +1,51 @@
+// Metro doesn't support TS and ES modules, so this file must be in plain JS and CommonJS
+const path = require('path');
+
+const { RELATIVE_PATH_REGEX, TREZOR_PACKAGES_SCOPES } = require('./constants');
+
+/**
+ * @typedef {Object} SecurityCheckParams
+ * @property {string} moduleName
+ * @property {string} originModulePath
+ */
+
+/**
+ * @param {SecurityCheckParams} params
+ * @returns {{isViolation: boolean, normalizedPath: string}}
+ */
+const checkSecurityViolation = ({ moduleName, originModulePath }) => {
+    const normalizedPath = path.normalize(moduleName);
+
+    const isNodeModules = originModulePath.includes('node_modules');
+    const isScopedPackage = TREZOR_PACKAGES_SCOPES.some(scope => normalizedPath.includes(scope));
+    const isRelativePath = RELATIVE_PATH_REGEX.test(normalizedPath);
```

Comment:
What `isRelativePath` mean in this context? Could you please provide an explanation comment here?

Reply:
added few comments